docs/reference/migration/migrate_8_0/packaging.asciidoc

```diff
@@ -15,3 +15,17 @@ Use Java 11 or higher. Attempts to run {es} 8.0 using earlier Java versions will
 fail.
 ====
 //end::notable-breaking-changes[]
+
+//tag::notable-breaking-changes[]
+.JAVA_HOME is no longer supported.
+[%collapsible]
+====
+*Details* +
+`JAVA_HOME` is no longer supported to set the path for the JDK. Instead, use
+the bundled JDK (preferable), or set `ES_JAVA_HOME`.
+
+*Impact* +
+Use the bundled JDK (preferable), or set `ES_JAVA_HOME`. `JAVA_HOME` will
+silently be ignored.
```

Comment:
This isn't true, the startup scripts will print a warning.

Reply:
Ah, would you be okay if we drop the word "silently" here?

Reply:
Works for me 👍

Reply:
I pushed 7ac980778b8c5e9041b34afa95c6b5d91ae697bb.